codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/Encoder.java

```diff
@@ -352,8 +352,7 @@ private void add(byte[] name, byte[] value) {
         }
 
         // Copy name and value that modifications of original do not affect the dynamic table.
-        name = Arrays.copyOf(name, name.length);
-        value = Arrays.copyOf(value, value.length);
+        // TODO: Do we need to copy ?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1135)